src/ripple/overlay/impl/OverlayImpl.h

```diff
@@ -25,6 +25,7 @@
 #include <ripple/basics/UnorderedContainers.h>
 #include <ripple/basics/chrono.h>
 #include <ripple/core/Job.h>
+#include <ripple/overlay/Message.h>
```

Comment:
We could. I don't really feel strongly about this, since this is under `impl` instead of an actual "public" header, but if you feel strongly about it, I'll make the change.

Reply:
I don't feel strongly about it.